antlr/src/main/antlr/FSH.g4

```diff
@@ -3,14 +3,17 @@ grammar FSH;
 options { tokenVocab = FSHLexer; }
 
 doc:                entity* EOF;
-entity:             alias | profile | extension | invariant | instance | valueSet | codeSystem | ruleSet | paramRuleSet | mapping;
+entity:             alias | profile | extension | invariant | instance | valueSet | codeSystem | ruleSet | paramRuleSet | mapping | logical | resource;
```

Comment:
b146f0c added `resource` but doesn't show any of the changes I'd expect in the ANTLR-generated files.  I think we need to re-generate those files.